minpy/numpy/mocking.py

```diff
@@ -67,17 +68,30 @@ def set_policy(self, policy):
         """[Deprecated] Set policy for this module."""
         self.generate_attrs(policy)
 
-    def generate_attrs(self, policy):
+    def record_op_stat(self):
+        """Record Op Calling Statistics"""
+        self.generate_attrs(self.policy, True)
+
+    def show_op_stat(self):
+        """Show Op Calling Statistics"""
+        self.policy.show_op_stat()
+
+    def generate_attrs(self, policy, use_selector=False):
```

Comment:
at line-73, i.e. when you're calling minpy.numpy.record_op_stat() before doing any calculation